### ISSUE TITLE
Add property "position" to FlxSprite

### ISSUE THREAD

**Issue by Dovyski** [FlixelCommunity/flixel#222]
There are several elements in `FlxObject` that have `x` and `y` properties, such as `velocity`, `acceleration`, `drag` and `maxVelocity` to name a few. They are all instances of `FlxPoint`, which totally makes sense.

The position of a `FlxObject`, however, is represented by two different `Number` properties. If they were represented by `FlxPoint` like everything else, checking the distance between two objetcs or sprites would be as easy as `a.position.distance(b.position)`.

I suggest adding a `position:FlxPoint` property to `FlxObject`, turning the current `x` and `y` properties into getters/setters.

**Comment by greysondn** [comment]
I will at least agree that I've found that property odd for some time.

The actual change is fairly simple, all things considered. Are we now 3.0 and okay with breaking API?

**Comment by Dovyski** [comment]
I was not planning to break the API because the `x` and `y` properties would still be there, both internally getting `position.x` and `posision.y`. 

Version 3.0 will definitely break the API since several classes have been moved into new packages. That's the reason why we bumped the version number from 2.58 to 3.0.

**Comment by greysondn** [comment]
... Deprecation warnings?

**Comment by Dovyski** [comment]
Yes, we introduced them for `3.0` to ensure a smoother transition. Take a look at [this](https://github.com/FlixelCommunity/flixel/blob/dev/src/flixel/FlxG.as#L1171), for instance.
